en/flight_modes/offboard.md

```diff
@@ -61,21 +61,26 @@ bool actuator
 ```
 
 The fields are ordered in terms of priority such that `position` takes precedence over `velocity` and later fields, `velocity` takes precedence over `acceleration`, and so on.
-The first field that has a non-zero value (from top to bottom) defines what valid estimate is required in order to use offboard mode, and the setpoint message(s) that can be used.
-For example, if the `acceleration` field is the first non-zero value, then PX4 requires a valid `velocity estimate`, and the setpoint must be specified using the `TrajectorySetpoint` message.
+The first field that has a non-zero value (from top to bottom) defines what valid estimate is required in order to use offboard mode, the setpoint message(s) that can be used, and the controllers that are enabled.
+For example, if the `acceleration` field is the first non-zero value, then PX4 requires a valid `velocity estimate`, the setpoint must be specified using the `TrajectorySetpoint` message `acceleration` field, and the position and velocity controllers are disabled (so that even if set, the setpoint position and velocity field will not be used).
 
 
 | desired control quantity | position field | velocity field | acceleration field | attitude field | body_rate field | actuator field | required estimate | required message                                                                                                                |
 |--------------------------|:--------------:|:--------------:|:------------------:|:--------------:|:---------------:|:---------------:|:-----------------:|---------------------------------------------------------------------------------------------------------------------------------|
-| position (NED)           |     &check;    |        -       |          -         |        -       |        -        |        -        |      position     | `TrajectorySetpoint`                                                                                                              |
-| velocity (NED)           |     &cross;    |     &check;    |          -         |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint`                                                                                                              |
-| acceleration (NED)       |     &cross;    |     &cross;    |       &check;      |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint`                                                                                                              |
+| position (NED)           |     &check;    |        -       |          -         |        -       |        -        |        -        |      position     | `TrajectorySetpoint.position`                                                                                                              |

```

Comment:
Here I added `TrajectorySetpoint.position`.
IFF the advice I give below that you should only set the setpoint fields matching the control quantity is good, then it makes sense to show the specific fields you can set. We would change the "required message" heading as well perhaps. 

For discussion :-)

Reply:
In my original intention I didn't specify the setpoint fields here but I put the specific setpoint structure in the section below, where all the possible combinations are explained (and in particular the field that must be set to `NaN` are introduced).
I fear that stating the fields here will lead users

1. to correctly set them
2. at the same to forget to put `NaN` to the others

I think it is better to force them to read the section below to understand the appropriate structure